pages/_app.js

```diff
@@ -83,6 +115,49 @@ export default function App({ Component, pageProps }) {
     categories.some((category) => filter.includes(category))
   );
 
+
+
+  useEffect(() => {
+  
+    if (searchTerm !== "") {
+      setTitle(FOUND_ACTIVITIES_TITLE);
+    } else {
+      setTitle(RANDOM_ACTIVITIES_TITLE);
+      setListedActivities(getRandomActivities());
+      return;
+    }
```

Comment:
Instead of using a state for the title, you can check this directly in the props: 

```jsx
title={searchTerm === '' ? "Activities You Might Like" : "Found Activities" }
```